pkg/audit/manager.go

```diff
@@ -41,6 +41,7 @@ var (
 	auditInterval             = flag.Uint("audit-interval", defaultAuditInterval, "interval to run audit in seconds. defaulted to 60 secs if unspecified, 0 to disable ")
 	constraintViolationsLimit = flag.Uint("constraint-violations-limit", defaultConstraintViolationsLimit, "limit of number of violations per constraint. defaulted to 20 violations if unspecified ")
 	auditFromCache            = flag.Bool("audit-from-cache", false, "pull resources from OPA cache when auditing")
+	detailedViolationsMetrics = flag.Bool("detailed-violations-metrics", false, "export detailed violations metrics")
```

Comment:
We should add the security/privacy risks to enabling these metrics to the description so users know the tradeoff they're making.